common/stats/stats.go

```diff
@@ -30,6 +30,8 @@ import (
 // For testing.
 var Time StatsTime = DefaultStatsTime()
 
+type UpTimeReportIntvl time.Duration
```

Comment:
making it its own type since it is injected by ice.  I'm thinking ice injected variables should be distinct types in case we need multiple variables of the same common type (though I'm not completely sure that this will prevent ice from confusing the multiple parameters)